students/youjin/westagram/users/views.py

```diff
@@ -1,3 +1,40 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
 
-# Create your views here.
+from .models import User
+
+import json
+import re
+
+class UserView(View):
+    def post(self, request):
+        data              = json.loads(request.body)
+        user_name         = data['name']
+        user_email        = data['email']
+        user_password     = data['password']
+        user_phone_number = data['phone_number']
+        user_gender       = data['gender']
+        user_sns          = data['sns']
+
+        try:
+            if not re.match('^([a-z0-9_\.-]+)@([\da-z\.-]+)\.([a-z\.]{2,6})$', user_email):  
+                return JsonResponse({"message": "EMAIL_ERROR"}, status=400)
+            
+            if User.objects.filter(email=user_email).exists():
+                return JsonResponse({"message" : "이미 존재하는 이메일입니다. 이메일을 다시 입력해주세요."}, status=400)
```

Comment:
에러 메세지는 실제 유저가 웹에서 볼 수 있는 에러가 아닙니다. api 서버에서 프론트엔드 개발자에게 보여줄 정보이기 때문에 너무 길게 작성할 필요가 없습니다.
```suggestion
                return JsonResponse({"message" : "email_already_exists"}, status=400)
```